guideline.md

```diff
@@ -0,0 +1,158 @@
+
+At any time you can check the process of your request on our [project board](https://github.com/orgs/PyLadiesGermany/projects/2).
+
+We expect chapters with multiple groups within it to have transparency with one another and to help us achieve an equitable way of distributing funds.
+
+### Request for larger amounts
+
+Some chapters might prefer to request approval for a larger amount in one go and manage the details of how it is spent internally (specifics still need to be reported and individual invoices sent). This could be for example be requesting a proportion of the total grant for the year upfront or for a single initiative which will include multiple costs. Ideally, such initiatives may be special programs that are indicated as planned by the chapter or chapters and presented, whenever possible, at the beginning of the year in order to plan the annual budget. After approval, the chapter will be able to get reimbursements for up to this amount without the need for further approval.
+
+These chapters should already be established, with a track record of events they have run (at least 5 events in the past 12 months) and (some) consistency in the organizing team (i.e. it should not be a whole new group who have recently taken over a chapter). It would be expected that this group would not receive approval for applications for further funds that year unless we have leftover budget (towards year end).
+
+The amount considered would be the total grant amount divided by the current number of active chapters + an additional slice for Germany as a whole. (i.e., if there are 4 active chapters and the grant is 5000 EUR a single chapter could request 1000EUR)
+
+**Process (differs slightly from above):**
+- Open a [Approval Multi Request](https://github.com/PyLadiesGermany/PySV_PyLadies_fund/issues/new?assignees=sleepypioneer%2Claysauchoa%2Cdiscombobulateme%2Cpga99%2Cterezaif&labels=approval_request%2Capproval_multi_request&projects=&template=approval-multi-request.md&title=Approval+Multi+Request) issue in this repository for the total amount, including ideas of what it will cover (exact items should be listed afterward)
+- Wait for approval for the total amount by at least 2 representatives outside of the requesting chapter
+- Individual invoices should be sent to the PySV with the PyLadies Germany email copied in
+- Reimbursement will happen within 14 days
+- Individual spends should be added to the ticket on the reporting platform marked as reimbursed with an update to the amount left
+- Add any additional information we can use for reporting after the funds are spent (optional but helps with reporting)
+- When the total amount has been reimbursed move the ticket to the reimbursed column. (consideration for further funds can then happen)
```

Comment:
is there a ticket in some board or do you mean to take some action on the issue?